lib/lib_div/ams/library.properties

```diff
@@ -0,0 +1,17 @@
+name=AMS Parser
+
+version=1.2.0
+
+author=Gunnar Skjold
+
+maintainer=https://github.com/gskjold
+
+sentence=Meter parsing Library for Espressif ESP32 and ESP8266 devices.
+
+paragraph=This library allows the ESP32 and ESP8266 devices to parse several meter protocolls
+
+category=Communication
+
+url=https://github.com/gskjold/AmsToMqttBridge

```

Comment:
The project was renamed and changed to https://github.com/UtilitechAS/amsreader-firmware